vcd/datasource_vcd_rde_type.go

```diff
@@ -0,0 +1,74 @@
+package vcd
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func datasourceVcdRdeType() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: datasourceVcdRdeTypeRead,
+		Schema: map[string]*schema.Schema{
+			"vendor": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The vendor name of the Runtime Defined Entity type",
+			},
+			"nss": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "A unique namespace associated with the Runtime Defined Entity type",
+			},
+			"version": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The version of the Runtime Defined Entity type. The version string follows semantic versioning rules",
```

Comment:
Just a thought: maybe we should add in the three Descriptions above that these three properties compose a single unique key?

Reply:
Done, also in `rde_interface`.